drivers/display/display_ili9340.c

```diff
@@ -18,13 +20,13 @@ LOG_MODULE_REGISTER(display_ili9340);
 #include <string.h>
 
 struct ili9340_data {
-#ifdef DT_INST_0_ILITEK_ILI9340_RESET_GPIOS_CONTROLLER
+#if DT_INST_NODE_HAS_PROP(0, reset_gpios)
 	struct device *reset_gpio;
 #endif
 	struct device *command_data_gpio;
 	struct device *spi_dev;
 	struct spi_config spi_config;
-#ifdef DT_INST_0_ILITEK_ILI9340_CS_GPIOS_CONTROLLER
+#if DT_INST_SPI_DEV_HAS_CS(0)
```

Comment:
Should this not read `DT_INST_SPI_DEV_HAS_GPIO_CS`?

Reply:
@mbolivar thoughts on the macro being `DT_INST_SPI_DEV_HAS_GPIO_CS`.  I can see the point of having `GPIO` in the name.